java/cloudbuild-release.yaml

```diff
@@ -30,10 +30,21 @@ steps:
     # go/airlock/howto_docker#example-gcb-dockerfile-in-docker-build-support
     id: buildx-create
     args: ["buildx", "build", "--add-host=metadata.google.internal:169.254.169.254",
-      "-t", "us-central1-docker.pkg.dev/$PROJECT_ID/release-images/java8", "."]
+      "--push",
```

Comment:
The entry in `images` should implicitly be doing the push (and registering the image for SBOM generation) unless `buildx` does something different

Reply:
It seems buildx is doing something different. https://screenshot.googleplex.com/8KEE6WcgXtqRuYe.png and https://stackoverflow.com/questions/78136934/how-to-tag-an-image-as-latest-using-docker-buildx